heron/executor/src/python/heron_executor.py

```diff
@@ -184,6 +186,16 @@ def __init__(self, args, shell_env):
     self.scheduler_classpath = parsed_args.scheduler_classpath
     self.scheduler_port = parsed_args.scheduler_port
     self.python_instance_binary = parsed_args.python_instance_binary
+    # self.heron_home = parsed_args.heron_home
```

Comment:
My first few comments still appear to be unaddressed. Is this github being wonky?